src/services/contracts/baseDAO/hooks/useLambdaAddPropose.ts

```diff
@@ -37,7 +37,7 @@ export const useLambdaAddPropose = () => {
 
         const data = await dao.proposeLambdaAdd(args, tezosToolkit)
 
-        mixpanel.track("Proposal Created", {
+        AnalyticsService.track("Proposal Created", {
```

Comment:
Another point that could be just my thinking, but by the time we're here at this point in the code the proposal has been added and it's already in our DB, so the anylicts should be later and before, so that we can see how many users initiated it and how many were actually successful. IF that's something we even wanna track @ashutoshpw @EightRice 

But this is completely optional.